mapApp/templates/mapApp/vis.html

```diff
@@ -9,67 +9,128 @@
 {% endblock %}
 
 {% block headerCSS %}
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dc/1.7.3/dc.css">
+  <link rel="stylesheet" href="https://unpkg.com/dc@4/dist/style/dc.css">
   <link rel="stylesheet" href="{% static 'mapApp/css/vis.css' %}">
 {% endblock %}
 
 {% block body %}
-  <div class="container" style="margin-top: 15px;">
-    <div class="row">
-      <div class="col-xs-12 col-sm-12 col-md-12">
-        <h4 class="text-info">{% trans "Click the charts to filter the data" %}</h4>
-        <div class="dc-data-count pull-left text-info">
-          {% blocktrans with '<span class="filter-count"></span>' as some and '<span class="total-count"></span>' as total %}{{some}} of {{total}} shown{% endblocktrans %}
+<div class="container" style="margin-top: 15px;">
+
+  <!--First Row: Title -->
+  <div class="row">
+    <div class="col-xs-12 col-sm-12 col-md-12">
+      <h3 class="vis-title">{% trans "Explore Incident Report Data" %}</h4>
+    </div>
+  </div>
+
+  <!--Second Row: Map & info box -->
+  <div class="row">
+    <!-- Map -->
+    <div class="col-xs-12 col-sm-12 col-md-7" style="height: 450px;">
+      <div id="map" class="center-chart"></div>
+    </div>
+    <div class="col-xs-12 col-sm-12 col-md-5">
+      <div id="vis-info" class="center-chart" style="height: 450px;">
+        <h4>
+          {% trans "Use this dashboard to explore the report data!" %}
+        </h4>
+        <p>
+          {% trans "Any filters you apply to the charts or the map will affect the output of all the visualizations." %}
+        </p>
+        <p>
+          {% trans "To get started, try this example query:" %}
+          <ol>
+            <li>{% trans "Zoom in to your city on the map" %}</li>
+            <li>{% trans "Select 'Collisions' and 'Nearmisses' on the total reports type chart" %}</li>
+            <li>{% trans "Highlight 'Mon-Fri' to view the workweek on the weekly chart" %}</li>
```

Comment:
Again, people might not find it obvious how to select a date range. Would it be possible to have the drag handles visible and show everything selected by default? Would we even want to do this?

Reply:
I spent a bit of time trying to find a way to always show the brush handles and couldn't find a solution. Do you want to look into it further? I think if it's giving people a lot of trouble another option could be to do a screen recording of someone using it and have that pop up in a little window?